### PR TITLE
Beacon-api: save atts to pool

### PR DESCRIPTION
**Pull request by terencechain**
When calling the beacon-API endpoint `SubmitAttestation`, we also should save submitted both types of attestations to the operation pool. This PR also removes go routine to use blocking. Reason being:
1.) Saving valid attestation only
2.) Saving an attestation to cache is cheap. Spinning up a go routine may cost more than that. If someone submits 10 attestations, then we'll spin up 10 go routines. I think blocking is better here

### CODE REVIEW

**Review by rauljordan — DISMISSED** [review]

File: beacon-chain/rpc/eth/beacon/pool.go [hunk 1]
```diff
@@ -115,6 +115,16 @@ func (bs *Server) SubmitAttestations(ctx context.Context, req *ethpbv1.SubmitAtt
...
+
+		if corehelpers.IsAggregated(att) {
+			if err := bs.AttestationsPool.SaveAggregatedAttestation(att); err != nil {
+				log.WithError(err).Error("could not save aggregated att")
+			}
+		} else {
+			if err := bs.AttestationsPool.SaveUnaggregatedAttestation(att); err != nil {
+				log.WithError(err).Error("could not save unaggregated att")
+			}
+		}
```
Comment (rkapka) [inline beacon-chain/rpc/eth/beacon/pool.go:127]: We already save in the goroutine on line 95. But we assume it's unaggregated. We also do this assuming the attestation is unaggregated. I don't know why this assumption was made. This is probably the actual thing to fix.
```
bs.OperationNotifier.OperationFeed().Send(&feed.Event{
		Type: operation.UnaggregatedAttReceived,
		Data: &operation.UnAggregatedAttReceivedData{
			Attestation: att,
		},
})
```

Comment (terencechain) [reply beacon-chain/rpc/eth/beacon/pool.go:127]: ah I missed that, I'll amend to that, thanks!

**Review by rkapka — CHANGES_REQUESTED** [review]
We should amend the function to save unaggregated and notify about them.

**Comment by terencechain** [comment]
@rkapka I ended up removing the go routine and defaulting this PR's behavior for these reasons

1. I think we should be saving valid attestation only
2. Saving an attestation to cache is cheap. Spinning up a go routine may cost more than that. If someone submits 10 attestations, then we'll spin up 10 go routines. I think blocking is better here

**Review by james-prysm — DISMISSED** [review]
LGTM

**Review by rkapka — APPROVED** [review]

**Pull request merged**